tests/ui/tox.ini

```diff
@@ -0,0 +1,11 @@
+# content of: tox.ini , put in same dir as setup.py
+[tox]
+envlist = ui-tests
+[testenv]
+basepython = python2.7
```

Comment:
I don't think this is needed, we should be able to let Tox pick the python version.